ark/utils/load_utils.py

```diff
@@ -249,36 +249,54 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
     return img_xr
 
 
-def load_imgs_from_dir(data_dir, imgdim_name='compartments', image_name='img_data', delimiter=None,
-                       dtype="int16", variable_sizes=False, force_ints=False):
-    """Takes a set of images from a directory and loads them into an xarray based on filename
-    prefixes.
+def load_imgs_from_dir(data_dir, files=None, delimiter=None, xr_dim_name='compartments',
+                       xr_channel_names=None, dtype="int16", force_ints=False,
+                       channel_indices=None):
+    """Takes a set of images (possibly multitiffs) from a directory and loads them
+     into an xarray.
 
     Args:
         data_dir (str):
             directory containing images
-        imgdim_name (str):
-            sets the name of the last dimension of the output xarray
-        image_name (str):
-            sets name of the last coordinate in the output xarray
+        files (list):
+            list of files (e.g. ['fov1.tif'. 'fov2.tif'] to load.
+            If None, all (.tif, .jpg, .png) files in data_dir are loaded.
         delimiter (str):
-            character used to determine the file-prefix containging the fov name. Default is None.
+            character used to determine the file-prefix containging the fov name.
+            Default is None.
+        xr_dim_name (str):
+            sets the name of the last dimension of the output xarray.
+            Default: 'compartments'
+        xr_channel_names (list):
+            sets the name of the coordinates in the last dimension of the output xarray.
         dtype (str/type):
             data type to load/store
-        variable_sizes (bool):
-            Dynamically determine image sizes and pad smaller imgs w/ zeros
         force_ints (bool):
             If dtype is an integer, forcefully convert float imgs to ints. Default is False.
+        channel_indices (list):
+            optional list of imgs to load, if None or empty loads all imgs
```

Comment:
This isn't the list of images, it's a list of indices specifying which images to keep. I think we should update this to make it more clear, since this is the only case where we use indices rather than names

Reply:
fixed